website/project/model.py

```diff
@@ -436,6 +436,8 @@ class NodeLog(StoredObject):
     RETRACTION_CANCELLED = 'retraction_cancelled'
     RETRACTION_INITIATED = 'retraction_initiated'
 
+    PREREG_CHALLENGE = 'prereg_challenge'
```

Comment:
I know you just picked up this PR, but can you explain the rationale for a separate log type? Is this just to support the frontend representation? 

Not a huge fan of using a separate log type (especially to override PROJECT_REGISTERED) to encode what is ultimately some basic metadata. What about instead use NodeLog.params to include something like `registration_schema`. This _should_ be  backwards compatible with old Prereg submissions (should we choose to migrate them), and pushes the logic for special representation of different registrations to the frontend.